semantic/src/site/globals/site.overrides

```diff
@@ -930,6 +929,7 @@ a:hover {
       margin-bottom: 5px;
       .ui.label {
         line-height: 1.5;
+        font-size: 0.57;
       }
       .centered-flex-box-row {
         display: flex;
```

Comment:
Why 0.57?  That's just what seemed to look best.

Reply:
![Screenshot 2023-11-07 at 12 05 40 PM](https://github.com/msupply-foundation/conforma-web-app/assets/5456533/e4c3370d-c265-4234-ad59-749989c64e7f)

I reckon we can lose the "STAGE" label, cos it's clear from the coloured bubble that it's referring to stage, which will give more room for the other stuff.

And maybe, let it wrap so the "Stage" bubble can go to next line? (It'd be good if the text/arrow inside the drop-down can stay on a single line)

Reply:
Text Label gone.  Left as much spacing btn dropdown and Stage Label as I could.  It's not much really.
![assignment_tab](https://github.com/msupply-foundation/conforma-web-app/assets/17306152/f1dde0ff-3229-4e7d-9dcc-5709e72b3d1a)